README.md

```diff
@@ -1,17 +1,19 @@
-# 𝑾𝒆𝒊𝒓𝒅 𝑭𝒐𝒏𝒕𝒔
-𝑨 𝑱𝒂𝒗𝒂𝑺𝒄𝒓𝒊𝒑𝒕 𝒑𝒂𝒄𝒌𝒂𝒈𝒆 𝒕𝒉𝒂𝒕 𝒕𝒖𝒓𝒏 𝒂𝒍𝒑𝒉𝒂𝒏𝒖𝒎𝒆𝒓𝒊𝒄 𝒄𝒉𝒂𝒓𝒂𝒄𝒕𝒆𝒓𝒔 𝒊𝒏𝒕𝒐 𝒘𝒆𝒊𝒓𝒅 𝒇𝒐𝒏𝒕 𝒔𝒕𝒚𝒍𝒆.
+# 𝖂𝖊𝖎𝖗𝖉 𝕱𝖔𝖓𝖙𝖘
```

Comment:
Hi, why change the font style?  I think this font style is hard to read.

Reply:
More classical or weirder ? but that's just my opinions 🎲

Reply:
Can you change it to original style?

Reply:
about Readme.md ?

Reply:
yes

Reply:
1

Reply:
That's fine. Thanks.